test/client/index_test.rb

```diff
@@ -381,79 +388,87 @@
       assert_equal id, percolator.id
     end
 
-    it "performs multi percolate queries" do
-      # COMPATIBILITY
-      if requires_percolator_mapping?
-        @index.update_mapping("percolator", { properties: { query: { type: "percolator" } } })
-      end
+    it "performs multi percolate queriess" do
```

Comment:
```suggestion
    it "performs multi percolate queries" do
```
Minor: removing an extra `s`

Reply:
Ah sorry I missed that!